Jenkinsfile

```diff
@@ -112,6 +114,12 @@ pipeline {
                 steps: this
                 )
               def mapPatallelTasks = pythonTasksGen.generateParallelTests()
+
+              // Let's now enable the windows stages
+              readYaml(file: '.ci/.jenkins_windows.yml')['windows'].each { v ->
+                def description = "${v.VERSION}${v.DISTUTILS_USE_SDK.equals('1') ? '-sdk' : ''}-${v.WEBFRAMEWORK}"
+                mapPatallelTasks["windows-${description}"] = generateStepForWindows(v)
+              }
               parallel(mapPatallelTasks)
```

Comment:
It's not related to this PR but I wouldn't mind at all if we took this chance to correct this misspelling and changed `mapPatallelTasks` to `mapParallelTasks` on this line, and L121 and L116. :)

Reply:
Did I say that I even rename the variable in one of the iterations I did run and I didn't even see it. Definitely I'm blind!! Good spot. I'll do it here then :)